### PR TITLE
Add Use Cases and edit QuickStart

### LINKED ISSUE #173: Some Use Cases we should add
1. Editing an elderly
2. Finding an elderly
3. Extending on 2 -> Wanting to mark/edit an eldelry with the a specific name (ie. combine use case 1 and 2)

### LINKED ISSUE #180: Small issue with Quick Start
image

Technically, 4.1 is wrong rite cuz there's sample data

### LINKED ISSUE #182: Make Quick Start Guide more user friendly

### PR DESCRIPTION
**Pull request by Zenkoh1**
Closes #182 , Closes #173

### CODE REVIEW

**Comment by GabrielCWT** [comment]
LGTM, just one thing I am unsure about. Should we be splitting the errors into separate errors? I'm unable to find anything regarding this on the CS2103T page.

I think the splitting of errors seems a bit unnecessary and bloats the use cases. Will hold off on approving and see what the others say.

**Review by GabrielCWT — COMMENTED** [review]
LGTM, just one thing I am unsure about. Should we be splitting the errors into separate errors? I'm unable to find anything regarding this on the CS2103T page.

I think the splitting of errors seems a bit unnecessary and bloats the use cases. Will hold off on approving and see what the others say.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -13,14 +13,41 @@ ContactMate is a **desktop app for managing clients at Active Ageing Centres (AA
...
+1. Open a command terminal for the folder you put the jar file in by following these steps (Folder is named `folder with contactmate jar` in the examples below) :
+   1. For Windows Users:
+      1. Right-click on the folder.
+      1. Click on `Open in Terminal` as highlighted below.
+        ![WindowsCmdOpen.png](images%2FWindowsCmdOpen.png)
+   1. For Mac Users:
+      1. Hold down the `Control` key and click on the folder (<kbd>Control</kbd> + <kbd>Click</kbd>).
+      1. Click on `New Terminal at Folder` as highlighted below.
+         ![MacCmdOpen.jpg](images%2FMacCmdOpen.jpg)
+1. In the command terminal you just opened in Step 5, type the `java -jar contactmate.jar` command and press Enter to run the application.<br>
```
Comment (brendanng7) [inline docs/UserGuide.md:50]: maybe can write type `java -jar contact mate.jar` and press Enter instead so its shorter and less confusing?

**Review by brendanng7 — COMMENTED** [review]
I get what Gabriel is saying but I think its just nice that the different errors and its expected behaviour doesn't have overlap. so splitting seems ok to me.

**Review by GabrielCWT — APPROVED** [review]

**Pull request merged**